### PR TITLE
Remove mongoDB from ROSPlan

### PR DESCRIPTION
**Pull request by oscar-lima**
- remove mongoDB from ROSPlan
- tested with rosplan_demos
- removed msgs that belong to rosplan_demos repo, e.g. CreatePRM.srv

### CODE REVIEW

**Comment by oscar-lima** [comment]
[rosplan demos mongoDB PR](https://github.com/KCL-Planning/rosplan_demos/pull/10) needs to be merged at the same time.

File: rosplan_planning_system/include/rosplan_action_interface/RPSimulatedActionInterface.h [hunk 1]
```diff
@@ -11,7 +11,7 @@
  * This file defines the RPMoveBase class.
  * RPMoveBase is used to connect ROSPlan to the MoveBase library.
  * PDDL "goto_waypoint" actions become "move_base_msgs::MoveBase" actions.
- * Waypoint goals are fetched by name from the SceneDB (implemented by mongoDB).
```
Comment (m312z) [inline rosplan_planning_system/include/rosplan_action_interface/RPSimulatedActionInterface.h]: Is this comment correct? Looks like it was copied from another class.

Comment (oscar-lima) [reply rosplan_planning_system/include/rosplan_action_interface/RPSimulatedActionInterface.h]: True.. I will remove it entirely. Thanks

File: rosplan_knowledge_msgs/CMakeLists.txt [hunk 2]
```diff
@@ -37,12 +36,9 @@ add_service_files(
   GetDomainNameService.srv
   GetDomainTypeService.srv
   GetDomainAttributeService.srv
-  GetDomainOperatorService.srv  
-  GetDomainOperatorDetailsService.srv  
-  GetDomainPredicateDetailsService.srv  
-  CreatePRM.srv
```
[the remark this answers is not in the document]
Comment (oscar-lima) [reply rosplan_knowledge_msgs/CMakeLists.txt]: Actually the only one that are removed are: CreatePRM.srv, AddWaypoint.srv and RemoveWaypoint.srv. If you see below (lines 39,40,41) git seems to be confused and is adding in green the same ones that are deleted for some reason (GetDomainOperatorService.srv, GetDomainOperatorDetailsService.srv, GetDomainPredicateDetailsService.srv)

File: rosplan_knowledge_msgs/package.xml [hunk 3]
```diff
@@ -3,28 +3,27 @@
...
+
   <maintainer email="michael.cashmore@kcl.ac.uk">michael</maintainer>
+  <maintainer email="oscar.lima@dfki.de">Oscar Lima</maintainer>
+
   <license>BSD</license>
   <url type="website">https://github.com/KCL-Planning/ROSPlan</url>
 
   <buildtool_depend>catkin</buildtool_depend>
-
-  <build_depend>message_generation</build_depend>
```
[the remark this answers is not in the document]
Comment (oscar-lima) [reply rosplan_knowledge_msgs/package.xml]: same as before. Git confusion, is adding ```<build_depend>message_generation</build_depend>``` in line 18 below.

**Pull request merged**